figures/tasks.py

```diff
@@ -328,6 +374,63 @@ def populate_daily_metrics_next(site_id=None, force_update=False):
                            site_count=sites_count))
 
 
+@shared_task
+def daily_metrics_callback(results, *args):
+    logger.info(results)
+    logger.info(args)
+
+
+@shared_task
+def populate_daily_metrics_async_sites(site_id=None, force_update=False):
+    """Next iteration to collect daily metrics for all sites in a deployment
```

Comment:
@OmarIthawi This docstring appears unchanged from `populate_daily_metrics_next`. Please provide context, lifting my comment from above:

Why this function? What problem does it solve? Is it for Tahoe only, can it run in standalone? What do you want to jog your memory if you have to revisit this function several months from now or hand off to another engineer? (Stuff that might not be crystal clear in the function source

To note, I should plan to remove the original `populate_daily_metrics` method and revamp the docstring for `populate_daily_metrics_next` to answer the same questions